src/components/ui/breadcrumb.tsx

```
@@ -1,116 +1,115 @@
-		tabIndex={-1}
-		className={cn("font-normal text-foreground", className)}
-		{...props}
-	/>
+  <span
+    ref={ref}
+    role="link"
+    aria-disabled="true"
+    aria-current="page"
+    className={cn("font-normal text-foreground", className)}
+    {...props}
+  />
```

Biome (2.1.2)

[error] The HTML element with the interactive role "link" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "link".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)